src/documentation/pages/components-wip/tabs/examples/_wait-animations-lumen-centurylink.vue

```diff
@@ -0,0 +1,124 @@
+    | URL. You can enable this behavior by setting the <code>waitForAnimations</code> option to <code>true</code>.
+
+  .div(slot='example')
+    ul.chi-tabs.chi-navigationExample.chi-customExample
+      li(
+        :class='index === 0 ? "-active" : ""',
+        v-for='(link, index) in tabLinks'
+      )
+        a(:href='`?tab=${index + 1}`') {{ link }}
+  <Wrapper :slot="`code-${exampleId}-${tab.id}-webcomponent`" v-for="tab in headTabs" :key="tab.id">
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="tab.codeSnippets.webComponent.code" class="html"></code> 
+    </pre>
+  </Wrapper>
+  <Wrapper v-for="tab in headTabs" :slot="`code-${exampleId}-${tab.id}-htmlblueprint`" :key="tab.id">
+    <JSNeeded />
+    <pre class="language-html">
+      <code v-highlight="tab.codeSnippets.htmlBlueprint.code" class="html"></code> 
+    </pre>
+  </Wrapper>
+</ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import { HeadTabsInterface } from '../../../../models/models';
+
+declare const chi: any;
+
+@Component({
+  data: () => {
+    return {
+      exampleTabs: [
+        {
+          disabled: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          active: true,
+          id: 'htmlblueprint',
+          label: 'HTML blueprint',
+        },
+      ],
+      tabLinks: Array(6).fill('Tab Link'),
+      headTabs: [
+        {
+          active: true,
+          id: 'enabled',
+          label: 'Enabled',
+          selectedItemId: 1,
+          codeSnippets: {
+            webComponent: {
+              code: ``,
+            },
+            htmlBlueprint: {
+              code: ``,
+            },
+          },
+        },
+        {
+          id: 'disabled',
+          label: 'Disabled',
+          selectedItemId: 1,
+          codeSnippets: {
+            webComponent: {
+              code: ``,
+              description: ``,
+            },
+            htmlBlueprint: {
+              code: ``,
+            },
+          },
+        },
+      ],
+      exampleId: 'wait-animations-lumen-centurylink',
+    };
+  },
+})
+export default class WaitAnimationsLumenCenturyLink extends Vue {
+  selectedTab: any;
+  tab: any;
+
+  _setCodeSnippets() {
+    this.$data.headTabs.forEach((headTab: any) => {
+      let tabLinks = '';
+      this.$data.tabLinks.forEach((tabLink: string, index: number) => {
+        tabLinks += `
+  <li ${index === 0 ? 'class="-active"' : ''}>
+    <a href="/>${tabLink}</a>
+  </li>`;
+      });
+
+      headTab.codeSnippets.htmlBlueprint.code = `<ul id="example__tabs_navigation_${headTab.id}" class="chi-tabs">${tabLinks}
+</ul>
+
+<script>
+const navigationElem = document.getElementById('#example__tabs_navigation_${headTab.id}');
+chi.navigation(
+  navigationElem,
+  {waitForAnimations: ${headTab.id === 'enabled' ? 'true' : 'false'}}
+);
+<\/script>
+`;
+    });
+  }
+  created() {
```

Comment:
Be careful with the separations between functions and variables in all examples, as in this case.

```suggestion
  }
  
  created() {
```